lib/active_file/base.rb

```diff
@@ -46,7 +46,7 @@ def extension
       protected :extension
 
       def actual_root_path
-        root_path  || Dir.pwd
+        root_path  || File.dirname(File.realpath(__FILE__))
```

Comment:
Would this work? `File.new(File.dirname(__FILE__)).realpath`

I'm assuming we want to use `File.dirname(__FILE__)` over `__dir__` for ruby version compatibility. too bad.